ons-core/ons-client/src/main/java/org/apache/rocketmq/ons/api/impl/rocketmq/ONSClientAbstract.java

```diff
@@ -74,16 +75,16 @@ public ONSClientAbstract(Properties properties) {
         this.sessionCredentials.updateContent(properties);
         if (this.sessionCredentials.getOnsChannel().equals(ONSChannel.ALIYUN) &&
             (null == this.sessionCredentials.getAccessKey() || "".equals(this.sessionCredentials.getAccessKey()))) {
-            throw new ONSClientException("please set access key");
+            throw new OMSRuntimeException("please set access key");
```

Comment:
As the default onsChannel was changed to "ALIYUN", Exception will be thrown if the access key or secret key is not set when connecting local broker.

Reply:
Do not use some "quirk comments" on exception, it would be better to conform to Imperative sentence while starting to begin with  the verb